utils/api-examples/src/get-code.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiPromise, WsProvider } from '@polkadot/api'
+import { types } from '@joystream/types'
+
+async function main() {
+  const provider = new WsProvider('ws://127.0.0.1:9944')
+
+  const api = await ApiPromise.create({ provider, types })
+
+  const currentBlockHash = await api.rpc.chain.getBlockHash(1)
+
+  console.log('getting code as of block hash', currentBlockHash.toString())
+
+  const substrateWasm = await api.query.substrate.code.at(currentBlockHash)
+
+  console.log(substrateWasm.toHex())
```

Comment:
This gives me just `0x` as output (but also happens in Pioneer via `Extrinsics` tab)

Reply:
Yes I was surprised by this as well. Does this have anything to do with the limit on the size of Vec you identified?

Reply:
While working on https://github.com/Joystream/joystream/pull/1177 I noticed that in the `augment-api.ts` auto-generated by `@polkadot/typegen` based on runtime metadata there is no `substrate` module at all.

So i did a little digging and found out that:
- This module is not part of the metadata retrieved via `api.rpc.state.getMetadata()` (even when I query our current testnet node)
- The same query also outputs `0x` on current Kusama
- It doesn't throw any errors (which happens in case of the `Vec` size limit issue that I ran into)